Package.swift

```diff
@@ -5,7 +5,7 @@ import PackageDescription
 let package = Package(
     name: "SnowplowTracker",
     platforms: [
-        .iOS(.v8)
```

Comment:
Can we remove the whole `platforms` section instead? Do you see any drawback?
I wouldn't force everyone to drop iOS 8 without releasing a major version of the tracker.
Removing the whole `platforms` section should retain iOS 8 as the deployment target for Xcode 11.
I've done a quick test and it seems to work for me but I would be happy to have your feedback.

Reply:
Thanks for the heads up!
I was thinking about removing it too. I briefly aligned it with `FMDB`. If removing the version constraint is also an option, I'd prefer that.